proto/kvrpcpb.proto

```diff
@@ -44,6 +44,7 @@ message Context {
     uint64 term = 5;
     CommandPri priority = 6;
     IsolationLevel isolation_level = 7;
+    bool no_cache = 8;
```

Comment:
I think `no_cache` is not a good name, `not_update_cache` is more precise.

Reply:
How about `fill_cache`? Or `cache_hint`?

Reply:
Good idea! noun is better name than verb.